### ISSUE TITLE
version 1.5.2 can't load frame of animation

### ISSUE THREAD

**Issue by WZBbiao** [airbnb/lottie-ios#279]
Before version can load the file, but can't load after update version 1.5.2. files have been given:
[归档.zip](https://github.com/airbnb/lottie-ios/files/1104513/default.zip)

**Comment by bellchen** [comment]
please show me your code.

**Comment by WZBbiao** [comment]
this is my code! In this website based on the demo of modified.(@"data" is my JSON file)

```
@property (nonatomic, strong) LOTAnimationView *lottieLogo;
    self.lottieLogo = [LOTAnimationView animationNamed:@"data"];
    self.lottieLogo.contentMode = UIViewContentModeScaleAspectFill;
    [self.view addSubview:self.lottieLogo];
    [self.lottieLogo play];
```
[Example.zip](https://github.com/airbnb/lottie-ios/files/1104772/Example.zip)

**Comment by bellchen** [comment]
nothing wrong in your code, but a issue in demo;
call [self.lottieLogo play]; in viewDidAppear, then will be ok;
```
//- (void)viewWillAppear:(BOOL)animated {
//  [super viewWillAppear:animated];
//  [self.lottieLogo play];
//}
- (void)viewDidAppear:(BOOL)animated{
    [super viewDidAppear:animated];
    [self.lottieLogo play];
}
```

**Comment by WZBbiao** [comment]
All right! Hope that corrective demo, But I don't understand why this problem caused by the specific reason. because In my project,A lot of code like the demo!

**Comment by bellchen** [comment]
the reason about this issue might be:
the lottieLogo view's frame is CGRectZero when call [self.lottieLogo play] in viewWillAppear;
and in viewDidAppear, lottieLogo get its correct frame;

**Comment by buba447** [comment]
Should be fixed in Lottie 2.0! If you still have a problem please create a new issue!
